apps/web/pages/api/book/event.ts

```diff
@@ -555,6 +558,13 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       }
 
       await sendRescheduledEmails({ ...evt, additionInformation: metadata });
+
+      // Reschedule reminders
+      for (const reminder of eventType.attendeeReminders) {
+        if (reminder.method === "SMS" && evt.reminderPhone) {
+          await scheduleSMSAttendeeReminder(evt, evt.reminderPhone, reminder);
+        }
+      }
```

Comment:
When rescheduling bookings, we can not update the scheduled message directly. We have to delete them and create new ones.